packages/react-vapor/src/components/actions/SecondaryActions.tsx

```diff
@@ -24,7 +24,7 @@ export const SecondaryActions: React.FunctionComponent<ISecondaryActionsProps> =
         .filter((action: IActionOptions) => action.separator || action.enabled || action.hideDisabled === false)
         // Filter out all separator that are preceded by another separator
         .filter((action: IActionOptions) => {
-            if (!action.separator || (action.separator && lastFilteredAction && !lastFilteredAction.separator)) {
+            if (!action?.separator || (action?.separator && lastFilteredAction && !lastFilteredAction.separator)) {

```

Comment:
if we only want to execute this block if there's an action present then wouldn't filtering out all falseys be better?

`.filter((action: IActionOptions) => action)`

Reply:
Removed, Its was a test and I forgot to remove